### PR TITLE
pg-ext: fix version() to include PostgreSQL

### PR DESCRIPTION
**Pull request by fantix**
Also fixed alias name of statically-evaluated FuncCall

### CODE REVIEW

File: edb/server/main.py [hunk 1]
```diff
@@ -439,7 +439,24 @@ async def run_server(
...
+        source += f" on {nice_date}"
+    if ver_meta["scm_revision"]:
+        source += f" from revision {ver_meta['scm_revision']}"
+        if source_date := ver_meta["source_date"]:
+            nice_date = source_date.strftime("%Y-%m-%dT%H:%MZ")
+            source += f" ({nice_date})"
+    if source:
+        extras.append(f", built{source}")
+    if ver_meta["target"]:
+        extras.append(f"for {ver_meta['target']}")
```
Comment (elprans) [inline edb/server/main.py:456]: What's the point of open-coding `get_version_line()` here?

Comment (fantix) [reply edb/server/main.py:456]: This is a partial revert of https://github.com/edgedb/edgedb/pull/5115, it was open-coding before #5115; but we can still keep the `get_version_line()` here.

Comment (elprans) [reply edb/server/main.py:456]: I would keep it

File: edb/pgsql/resolver/static.py [hunk 2]
```diff
@@ -113,8 +115,21 @@ def eval_FuncCall(
 
     if fn_name == 'version':
         from edb import buildmeta
+
+        args = ["EdgeDB", buildmeta.get_version_string()]
```
Comment (elprans) [inline edb/pgsql/resolver/static.py:119]: Why not keep using `get_version_line` here also?

**Review by aljazerzen — APPROVED** [review]
Fivetran is satisfied, thanks.

**Pull request merged**